site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2,6 +2,7 @@
 
 namespace app\libraries\database;
 
+use _HumbugBoxec8571fe8659\Nette\Utils\DateTime;
```

Comment:
What is this for?

Reply:
Definitely not supposed to be there. Will remove that.